diff --git a/src/libbluray/bdj/bdj.c b/src/libbluray/bdj/bdj.c
index ef82ee4..c08c549 100644
--- a/src/libbluray/bdj/bdj.c
+++ b/src/libbluray/bdj/bdj.c
@@ -203,6 +203,7 @@ static void *_load_jvm(const char **p_java_home)
     static const char *jvm_path[] = {NULL, JDK_HOME,
                                      "/usr/lib/jvm/default-java",
                                      "/usr/lib/jvm/default",
+                                     "/usr/lib/jvm/",
                                      "/etc/java-config-2/current-system-vm",
                                      "/usr/lib/jvm/java-7-openjdk",
                                      "/usr/lib/jvm/java-8-openjdk",
